### PR TITLE
update sortables

### LINKED ISSUE #106: Sortables
Meeting 2021-03-08: @cportele to update the section on the sortables resource and align it with the one on [queryables](https://portal.ogc.org/files/?artifact_id=96288&version=2#filter-queryables) (JSON Schema that simply list the properties of the record that can be used for sorting; the recommended JSON Schema profile is even simpler as any information beyond the type is not needed).

### PR DESCRIPTION
**Pull request by cportele**
closes #106

### CODE REVIEW

**Review by pvretano — APPROVED** [review]

File: core/standard/clause_9_sorting.adoc [hunk 1]
```diff
@@ -18,24 +18,27 @@ handling missing values; specifying a high value indicating that the correspondi
...
 
-include::requirements/sorting/REQ_sortables-op.adoc[]
+include::requirements/sorting/REQ_get-sortables-op.adoc[]
 
-include::requirements/sorting/REQ_sortables-success.adoc[]
+include::requirements/sorting/REQ_get-sortables-success.adoc[]
+
+The response is returned as a JSON Schema document that describes a single JSON object where each property is a sortable. Note that the sortables schema does not specify a schema of any object that can be retrieved from the API. JSON Schema is used for the sortables to have a consistent approach for describing schema information and JSON Schema is/will be used in other parts of OGC API Features to describe schemas for GeoJSON feature content including in OpenAPI documents.
+
+NOTE: The OGC Feature API Standards Working Group has a schema harmonization task that could lead to a future OGC API standard that will deprecate the approach for the sortables resource specified in this document.
```
Comment (tomkralidis) [inline core/standard/clause_9_sorting.adoc:27]: `s/OGC Feature API Standards Working Group/OGC API - Features Standards Working Group/g` (sorry, I am fresh off a deep editorial sweep of the standard).

Comment (cportele) [reply core/standard/clause_9_sorting.adoc:27]: The name of the group is "Features API SWG", so the current text is *almost* correct (an "s" is missing). I will add it.

Comment (cportele) [reply core/standard/clause_9_sorting.adoc]: ... here is the link to the group: https://www.ogc.org/projects/groups/featapiswg

Comment (tomkralidis) [reply core/standard/clause_9_sorting.adoc]: OK.  Looks like the other OGC API Standards used the `OGC API - xxx` convention for their SWG names.  All good/carrying on.

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**